src/client/components/App.tsx

```diff
@@ -2,56 +2,19 @@ import * as React from "react";
 import TypeEditor from "./type/TypeEditor";
 import TypeList from "./type/TypeList";
 import Navbar from "./navbar/Navbar";
-
 import "./App.less";
-import { Type } from "../models/Type";
-import TypesStore from "./stores/TypesStore";
-
-
-interface AppState {
-    types: Type[]
-}
-
-// const getStateFromFlux = () => {
-//     return {
-//         isLoading: TypesStore.isLoading(),
-//         types: TypesStore.getTypes()
-//     }
-// }
 
-export default class App extends React.Component<{}, AppState> {
-    constructor(props: {}) {
-        super(props);
-        this.state = {
-            types: [
-                {
-                    name: "Test",
-                    description: "Description1"
-                },
-                {
-                    name: "Test",
-                    description: "Description1"
-                }
-            ]
-        }
-    }
-    handleTypeAdd = (newType: Type) => {
-        this.setState({
-            types: [...this.state.types, newType]
-        });
-    };
 
-    render() {
-        return (
-            <div>
-                <Navbar></Navbar>
+export default function App(props) {
```

Comment:
where extends React.Component ?

Reply:
Do you sow how to make component-container in the Facebook repo?

Reply:
No, I didn't, can you explain ?

Reply:
You can just make a view and pass all functional to it by Container.createFunctional.
All functional are our store and actions with which you can control the state of a view.

Reply:
https://github.com/facebook/flux/tree/65724d5c0f08359f976a312c5581fdb99a799983/examples/flux-todomvc/src

Reply:
Ok, but you describe props where his interface?